keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/vetos/RequiredLoadBalancerVeto.kt

```diff
@@ -41,7 +42,8 @@ class RequiredLoadBalancerVeto(
     } else {
       deniedResponse(
         message = missingLoadBalancers.joinToString(separator = "\n", transform = MissingDependency::message),
-        vetoArtifact = false
+        vetoArtifact = false,
+        suggestedStatus = MISSING_DEPENDENCY
```

Comment:
Now the `RequiredLoadBalancerVeto` and the `RequiredSecurityGroupVeto` both set the correct status, which will propagate through